pkg/drivers/kic/oci/oci.go

```diff
@@ -34,6 +34,30 @@ import (
 	"strings"
 )
 
+// DeleteAllContainersByLabel deletes all containers that have a specific label
+// if contatiners founds will not return an error
+// example: docker volume prune -f --filter label=name.minikube.sigs.k8s.io=minikube
+func DeleteAllContainersByLabel(ociBin string, label string) []error {
+	var deleteErrs []error
+	if ociBin == Docker {
+		if err := PointToHostDockerDaemon(); err != nil {
+			return []error{errors.Wrap(err, "point host docker-daemon")}
+		}
+	}
+	cs, err := listContainersByLabel(ociBin, label)
+	if err != nil {
+		glog.Infof("error listing containers by label %q: %v", label, err)
+	}
+	for _, c := range cs {
+		cmd := exec.Command(ociBin, "rm", "-f", "-v", c)
+		if out, err := cmd.CombinedOutput(); err != nil {
+			glog.Infof("error deleting container %s: output: %s", c, string(out))
```

Comment:
Either log the error, or return it. 

Anytime a function does both, it results in duplicate error messages being sent to the log.